app/views/spree/products/_detailed_description.html.erb

```diff
@@ -0,0 +1,90 @@
+<ul class="nav nav-tabs" role="tablist">
+  <li class="nav-item" role="presentation"><a class="nav-link active" data-toggle="tab" href="#description">Description</a></li>
+  <li class="nav-item" role="presentation"><a class="nav-link" data-toggle="tab" href="#howtouse">How to Use</a></li>
+  <li class="nav-item" role="presentation"><a class="nav-link" data-toggle="tab" href="#applications">Applications</a></li>
+  <li class="nav-item" role="presentation"><a class="nav-link" data-toggle="tab" href="#documents">Documents</a></li>
+</ul>
+<div class="tab-content">
+  <div class="tab-pane fade show active" id="description" role="tabpanel">
+    <div class="tab-content-wrap" id="product-description-long"  data-hook="description">
+      <%= sanitize product_description(@product) %>
```

Comment:
Please add this check (<% if Spree::Config[:show_raw_product_description] || product_wysiwyg_editor_enabled? %>) above long description. As its present in default spree.